.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,8 +1,11 @@
 blank_issues_enabled: true
 contact_links:
-  - name: xk6-browser Community Forum
-    url: https://community.k6.io/c/xk6-browser/14
+  - name: Community Forum
+    url: https://community.grafana.com/c/grafana-k6/k6-browser/79
     about: Please ask and answer questions here.
-  - name: xk6-browser Documentation
+  - name: Documentation
     url: https://github.com/grafana/k6-docs
     about: Please add any documentation related issues here.
+  - name: Migration Guide
+    url: https://k6.io/docs/using-k6-browser/migrating-to-k6-v0-46/
+    about: Please review the guide if you're migrating to k6 v0.46+ from an older version of xk6-browser.
```

Comment:
I'm not sure about saying `k6 v0.46+` as we're on the xk6-browser repo. Any suggestions?

Reply:
I think this is OK. I would not expect too many users being aware of xk6-browser version and using it as an extension now that it's bundled with k6. So I think referencing k6 version directly is probably the best :+1: